src/Analysis.R

```diff
@@ -5,24 +5,14 @@ library(forcats)
 library(stringr)
 library(tidyr)
 library(labelled)
-# library(gtsummary) ## TODO: Install {gtsummary} if necessary
-
-## ----file-system-objects----
-
-# TODO: Change "<dataset_filename>" by the name of your own dataset, if
-#   necessary (e.g., "dat/penguins.csv")
-DATASET_PATH <- "dat/<dataset_filename>.csv"
```

Comment:
Agree with adding it to the call to `read_csv()`, but I'd rather leave the wildcard, for two reasons:

- In case any of them prefers to use a dataset of their own with a different name (which we actually encourage).
- To "force" at least one change to commit.

What do you think?

Reply:
Ok!